library/HelloSign/Test/AbstractTest.php

```diff
@@ -50,13 +50,9 @@ protected function setUp()
         $api_url = $_ENV['API_URL'] == null ? Client::API_URL : $_ENV['API_URL'];
         $oauth_token_url = $_ENV['OAUTH_TOKEN_URL'] == null ? Client::OAUTH_TOKEN_URL : $_ENV['OAUTH_TOKEN_URL'];
         $this->client = new Client($_ENV['API_KEY'], null, $api_url, $oauth_token_url);
-        $this->team_member_1 = rand(1, 10000000) . "@example.com";
-        $this->team_member_2 = rand(1, 10000000) . "@example.com";
+        $this->team_member_1 = "sdk-test-php+test1@hellosign.com";
+        $this->team_member_2 = "sdk-test-php+test2@hellosign.com";
         // $this->client->enableDebugMode();
 
-        if ($api_url != Client::API_URL) {
-            $this->client->disableCertificateCheck();
```

Comment:
This function was previously removed.  Turns out we didn't need it.  However we left references to it around which was causing issues in some cases.

Reply:
@ericbecking so we will need to work on removing this function again AND all its references in the future, right? In general, seems harmless to keep it for now.

Reply:
To clarify, the function disableCertificateCheck(); does not currently exist in our codebase.  It was removed over a year ago.  As part of THIS PR I'm removing references to it so that we don't attempt to call a function that doesn't exist, which then throws a fatal error.